### PR TITLE
View parks info

### PR DESCRIPTION
**Pull request by libbypeterson**

### CODE REVIEW

File: src/main/ui/src/components/ParkView/ParkProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  return (
+    <div className="overall-div">
+      <ProfileHeroImage parkInfo={parkInfo} />
+      <Container className="park-description">
+        <div> {parkInfo.description} </div>
+      </Container>
+      <VisitedParkSwitch />
+
+      <ThingsToDoAccordion parkCode={parkCode} parkInfo={parkInfo} />
```
Comment (jaylingelbach) [inline src/main/ui/src/components/ParkView/ParkProfile.jsx:34]: Are we using the accordian? I don't see one. If not please remove this code.

File: src/main/ui/src/components/ParkView/ThingsToDoAccordion.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { React, useEffect, useState } from "react";
```
Comment (jaylingelbach) [inline src/main/ui/src/components/ParkView/ThingsToDoAccordion.jsx:1]: If not using accordian please remove.

File: src/main/ui/src/components/ParkView/VisitedParkSwitch.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (jaylingelbach) [inline src/main/ui/src/components/ParkView/VisitedParkSwitch.jsx:1]: nice!

**Review by jaylingelbach — CHANGES_REQUESTED** [review]

File: src/main/ui/src/components/ParkView/ParkProfile.jsx (hunk 1 above)
Comment (emkharr) [reply src/main/ui/src/components/ParkView/ParkProfile.jsx:34]: Accordion is being used to display activities/hikes in the park

**Review by jaylingelbach — APPROVED** [review]

**Pull request merged**